new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAddCluster.java

```diff
@@ -473,12 +448,15 @@ public void testAddMiiConfiguredCluster() {
     String configMapName = "configclusterconfigmap";
     createClusterConfigMap(configMapName, "model.config.cluster.yaml");
 
-    // get the creation time of the admin server pod before patching
-    String adminPodCreationTime = getadminPodCreationTime();
+    LinkedHashMap<String, String> pods = new LinkedHashMap<>();
 
+    // get the creation time of the admin server pod before patching
+    String adminPodCreationTime = getPodCreationTime(domainNamespace, adminServerPodName);
```

Comment:
The code that get the creation timestamp of all servers and add it to a map is duplicated in the test cases.
Consider make it a common test util method?

Reply:
Done. Added method(s) to utils/CommonTestUtils.java